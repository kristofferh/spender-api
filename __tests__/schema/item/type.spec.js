import ItemType from "schema/item/type";

describe("Item Type", () => {
  it("Should match snapshot", () => {
    expect(ItemType.getFields()).toMatchSnapshot();
  });
});
