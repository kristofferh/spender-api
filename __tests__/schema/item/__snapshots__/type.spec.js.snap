// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item Type Should match snapshot 1`] = `
Object {
  "email": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "email",
    "type": "String",
  },
  "id": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "id",
    "type": "Int",
  },
  "items": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "items",
    "resolve": [Function],
    "type": "[Item]",
  },
  "tags": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "tags",
    "resolve": [Function],
    "type": "[Tag]",
  },
  "token": Object {
    "args": Array [],
    "isDeprecated": false,
    "name": "token",
    "type": "String",
  },
}
`;
